src/cljs/kefirnadar/application/views.cljs

```diff
@@ -177,19 +193,31 @@
         [:option {:value ""} "Izabarite opstinu"]
```

Comment:
typo

Reply:
Hahahahahah izabarite, to stoji od pocetka 🤣👍

Reply:
Video sam i ovaj gore komentar, ima dosta da se sljaka, svestan sam, jako sam loš za dizajn, potrudicu se da sve radi kako treba al za lepotu sam slep sto se kaze..
Znam za linkove, samo sam stavio dummy data na ta mesta pa da odlucim/o sta ce tu biti